object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/FrameworkSupport.java

```diff
@@ -0,0 +1,39 @@
+package org.checkerframework.checker.objectconstruction.framework;
+
+import com.sun.source.tree.NewClassTree;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+
+/** add CalledMethods annotations to code generated by framework */
+public interface FrameworkSupport {
+
+  /**
+   * Allows for supporting a framework's {@code toBuilder} routine. Typically, the returned Builder
+   * has had all of its required setters invoked. So, the framework support can add a {@link
+   * org.checkerframework.checker.objectconstruction.qual.CalledMethods} annotation capturing this
+   * fact.
+   *
+   * @param t a method whose name is "toBuilder". An implementation must check that this is actually
```

Comment:
I still don't think this is true. See the call-site in the ATF: https://github.com/kelloggm/object-construction-checker/pull/112/files#diff-c9157a43115d763be3bd781779fbca90R307

`t` is *not* guaranteed to be named `toBuilder`, and this Javadoc should not suggest that it will be. In fact, this method will be called for every executable element; it is incumbent on the framework support implementation to do *all* the checking necessary to conclude that `t` is a `toBuilder` method, including checking its name.

Reply:
Yes, you're right.  The logic changed since I looked before.  Will fix